src/mocks/repositories/releaseRecipientRepository.ts

```diff
@@ -12,7 +12,7 @@ const seeds: Array<ReleaseRecipientFieldsFragment> = [
   releaseRecipientFactory.build({ username: 'lh7', fullName: 'Liam Hickey' }),
   releaseRecipientFactory.build({ username: 'vk8' }),
   releaseRecipientFactory.build({ username: 'cc36' }),
-  releaseRecipientFactory.build({ username: 'aw24' }),
+  releaseRecipientFactory.build({ username: 'aw24', fullName: 'Andrew Wiles' }),
```

Comment:
What is this change about in this context?

Reply:
The release tests keep randomly failing as it is not able to find the full user in the select list, as it is used to be in the bottom, I just used to a user name that can be at the top (as the list is alphabetic sorted) to stop the release tests from failing.